recipes/graphclust-wrappers/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: graphclust-wrappers
+  version: '0.1'
+
+source:
+  fn: 0.7.6-galaxy.tar.gz
+  url: https://github.com/BackofenLab/GraphClust/archive/0.7.6-galaxy.tar.gz
+  sha256: c8eca5d2d0f9f7140e82cc16beb1e4d5cb784eb9a467061090f7430ee2878846
+  # git_url: https://github.com/BackofenLab/GraphClust.git
+  # git_rev: 0.7.6
+
+build:
+  number: 0
+  skip: True  # [not linux64 or osx]
+
+requirements:
+  build:
+    - perl-math-round
+    - perl-array-utils
+
+  run:
+    - perl-math-round
+    - perl-array-utils
+
+test:
+  commands:
+    ls; ./LocARNAGraphClust/mloc2stockholm.pl -h 2>&1 | grep "USAGE" 
```

Comment:
This needs to be a list in yaml.
This should work ideally ... (only if your script are directly in your PREFIX/bin)
```
- mloc2stockholm.pl -h 2>&1 | grep "USAGE" 
```